### ISSUE TITLE
Search placeholder does not meet contrast requirements in Firefox

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6908]
ref #6144 

## Environment

> Operating system

Windows 10

> Browser

Firefox 80

> Automated testing tool and ruleset

Colour Contrast Analyzer

> Assistive technology used to verify

na

## Detailed description

Firefox isn't picking up our styling for search's placeholder (which should be `$text-05`

### Firefox
![image](https://user-images.githubusercontent.com/40970507/94202798-f4d51e80-fe83-11ea-8a43-499570f335e4.png)

### Edge
![image](https://user-images.githubusercontent.com/40970507/94202835-028aa400-fe84-11ea-8dd6-9553e182b4ff.png)

### Chrome
![image](https://user-images.githubusercontent.com/40970507/94202845-07e7ee80-fe84-11ea-8a74-74a92b433880.png)
